Formula/hey.rb

```diff
@@ -0,0 +1,27 @@
+class Hey < Formula
+  desc "Load testing for the 21st century"
+  homepage "https://github.com/rakyll/hey"
+  url "https://github.com/rakyll/hey.git",
+    :tag => "v0.1.2",
+    :revision => "01803349acd49d756dafa2cb6ac5b5bfc141fc3b"
+  head "https://github.com/rakyll/hey.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "on"
+
+    dir = buildpath/"src/github.com/rakyll/hey"
+    dir.install buildpath.children
+
+    cd dir do
+      system "go", "build", "-o", bin/"hey"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system "#{bin}/hey https://google.com"
```

Comment:
I might need some on concatenating the multiple lines into one line, so that i can assert 

`[200]	200 responses`